lib/ic/microcontroller/stm32f0xxgxux.yaml

```diff
@@ -0,0 +1,35 @@
+datasheet: http://www.st.com/resource/en/datasheet/stm32f042g6.pdf 
+description: STM32F0 28 pin UQFPN package
+footprint: agg:QFN-28-ST
+name: STM32F0xxGxUx
+ordercodes:
+- [Farnell, '2469551']
+pins:
+- - - [VDD, 17, pwrin]
+    - [VDDIO2, 18, pwrin]
+    - [VDDA, 5, pwrin]
+    - [VSS, 16, pwrin]
+  - - [NRST, 4, in]
+    - [BOOT0, 1, in]
+  - - [PF0, 2, bidi]
+    - [PF1, 3, bidi]
+  - - [PB0, 14, bidi]
+    - [PB1, 15, bidi]
+    - [PB3, 24, bidi]
+    - [PB4, 25, bidi]
+    - [PB5, 26, bidi]
+    - [PB6, 27, bidi]
+    - [PB7, 28, bidi]
+- - - [PA0, 6, bidi]
+    - [PA1, 7, bidi]
+    - [PA2, 8, bidi]
+    - [PA3, 9, bidi]
+    - [PA4, 10, bidi]
+    - [PA5, 11, bidi]
+    - [PA6, 12, bidi]
+    - [PA7, 13, bidi]
+    - [PA9, 19, bidi]
+    - [PA10, 20, bidi]
```

Comment:
Please indicate that PA9 and PA10 can also be PA11 and PA12, probably `PA9-PA11` and `PA10-PA12`.